cookbook/deployment/fortrabbit.rst

```diff
@@ -0,0 +1,227 @@
+      $container->setParameter('database_host', $secrets['MYSQL']['HOST']);
+      $container->setParameter('database_name', $secrets['MYSQL']['DATABASE']);
+      $container->setParameter('database_user', $secrets['MYSQL']['USER']);
+      $container->setParameter('database_password', $secrets['MYSQL']['PASSWORD']);
+   }
+
+   // Check if the Memcache component is present
+   if (isset($secrets['MEMCACHE'])) {
+        
+      $memcache = $secrets['MEMCACHE'];
+      $handlers = [];
+        
+      foreach (range(1, $memcache['COUNT']) as $num) {
+         $handlers [] = $memcache['HOST'. $num]. ':'. $memcache['PORT'. $num];
+      }
+
+      // Apply ini settings
+      ini_set('session.save_handler', 'memcached');
+      ini_set('session.save_path', implode(',', $handlers));
+      
+      if ($memcache['COUNT'] == 2) {
+         ini_set('memcached.sess_number_of_replicas', 1);
+         ini_set('memcached.sess_consistent_hash', 1);
+         ini_set('memcached.sess_binary', 1);
+      }
+      
+   }
+
+
+Make sure this file is listed in your *imports*:
+
+.. code-block:: yaml
+
+   # app/config/config_prod.yml
+   imports:
+      - { resource: config.yml }
+      - { resource: config_prod_secrets.php }
+   
+   # ..
+   framework:
+      session:
+         # set handler_id to null to use default session handler from php.ini (memcached)
+         handler_id:  ~
+   # ..    
+
+
+
+Configuring the Environment in the Dashboard
+--------------------------------------------
+
+PHP Settings
+~~~~~~~~~~~~
+
+The PHP version and enabled extensions are configuable under the PHP settings 
+of your App within the fortrabbit Dashboard.
+
+
+Environment Variables
+~~~~~~~~~~~~~~~~~~~~~
+
+Set the `SYMFONY_ENV` environment variable to `prod` to make sure the right 
+config files get loaded. ENV vars are configuable in fortrabbit Dashboard as well.
+
+
+Document Root
+~~~~~~~~~~~~~
+
+The document root is configuable for every custom domain you setup for your App. 
+The default is /htdocs, but for Symfony you probalby want to change it to 
+/htdocs/web. You also do so in the fortrabbit Dashboard under `Domain` settings.
+
+
+Deploying to fortrabbit
+-----------------------
+
+It is assumed that your codebase is under version-control with Git and dependencies
+ are managed with Composer (locally).
+
+Every time you push to fortrabbit composer install runs before your code gets 
+deployed. To finetune the deployment behaviour put a `fortrabbit.yml`_. deployment 
+file (optional) in the project root.
+
+Add fortrabbit as a (additional) Git remote and add your configuration changes.
+
+.. code-block:: bash
+
+   $ git remote add fortrabbit git@deploy.eu2.frbit.com:your-app.git
+   $ git add composer.json composer.lock
+   $ git add app/config/config_prod_secrets.php
+
+Commit and push
+
+.. code-block:: bash
+
+   $ git commit -m 'fortrabbit config'
+   $ git push fortrabbit master -u
+
+**Note:** Replace your-app with the name of your fortrabbit App.
```

Comment:
The syntax for notes in RST files is a bit different.

Instead of:

```
**Note:** Replace your-app with the name of your fortrabbit App.
```

use this:

```
.. note::

    Replace your-app with the name of your fortrabbit App.
```